lib/cookies.js

```diff
@@ -0,0 +1,33 @@
+var util = require('./util');
+
+function setCookie(name, value, expiresAt) {
+  var expiresText = '';
+  if (expiresAt) {
+    expiresText = ' expires=' + util.isoToDate(expiresAt).toUTCString() + ';';
```

Comment:
We don't use isoToDate anywhere else. I think I wrote this at a time when I wasn't sure what was necessary for OAuth. I can consolidate it.